src/open_inwoner/accounts/views/contacts.py

```diff
@@ -41,7 +41,12 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
         user = self.request.user
-        context["pending_approvals"] = user.get_contacts_for_approval()
+        pending_approvals = User.objects.get_pending_approvals(user)
+        if pending_approvals.count() == 1:
+            context["pending_approval"] = pending_approvals.get()
+        else:
+            context["pending_approval_list"] = pending_approvals
```

Comment:
it looks excessive. Why not to have just `context["pending_approval_list"] = pending_approvals` and check the count in the template?